tests/test_venue_request.py

```diff
@@ -1234,25 +1242,67 @@ def test_venue_decision_stage(self, client, test_client, selenium, request_page,
+                    'suggestions': {
+                        'value-regex': '[\\S\\s]{1,5000}',
+                        'description': 'Please provide suggestions on how to improve the paper',
+                        'required': False,
+                    }
+                },
+                'upload_decisions': url
+            },
+            forum=venue['request_form_note'].forum,
+            invitation='{}/-/Request{}/Decision_Stage'.format(venue['support_group_id'],
+                                                              venue['request_form_note'].number),
+            readers=['{}/Program_Chairs'.format(venue['venue_id']), venue['support_group_id']],
+            referent=venue['request_form_note'].forum,
+            replyto=venue['request_form_note'].forum,
+            signatures=['~SomeFirstName_User1'],
+            writers=[]
+        ))
+
+        assert decision_stage_note
+        helpers.await_queue()
+
+        process_logs = client.get_process_logs(id=decision_stage_note.id)
+        assert len(process_logs) == 1
+        assert process_logs[0]['status'] == 'ok'
+
+        for sub in submissions:
+            sub_decision_note = test_client.get_notes(
+                invitation='{venue_id}/Paper{number}/-/Decision'.format(
+                    venue_id=venue['venue_id'], number=sub.number
+                )
+            )[0]
+            assert sub_decision_note
+
         # Post a decision note using pc test_client
         program_chairs = '{}/Program_Chairs'.format(venue['venue_id'])
         area_chairs = '{}/Paper{}/Area_Chairs'.format(venue['venue_id'], submission.number)
         senior_area_chairs = '{}/Paper{}/Senior_Area_Chairs'.format(venue['venue_id'], submission.number)
-        decision_note = test_client.post_note(openreview.Note(
-            invitation='{}/Paper{}/-/Decision'.format(venue['venue_id'], submission.number),
-            writers=[program_chairs],
-            readers=[program_chairs, senior_area_chairs, area_chairs],
-            nonreaders=['{}/Paper{}/Authors'.format(venue['venue_id'], submission.number)],
-            signatures=[program_chairs],
-            content={
-                'title': 'Paper Decision',
-                'decision': 'Accept',
-                'comment':  'Good paper. I like!',
-                'suggestions': 'Add more results for camera ready.'
-            },
-            forum=submission.forum,
-            replyto=submission.forum
-        ))
+        decision_note = test_client.get_notes(
+            invitation='{venue_id}/Paper{number}/-/Decision'.format(
+                venue_id=venue['venue_id'], number=submission.number
+            )
+        )[0]
+
+        decision_note.content = {
+            'title': 'Paper Decision',
+            'decision': 'Accept',
+            'comment':  'Good paper. I like!',
+            'suggestions': 'Add more results for camera ready.'
+        }
+        decision_note = test_client.post_note(decision_note)
```

Comment:
shouldn't be the decision note posted by the process function?

Reply:
here you need to check if decision notes are posted and they match with the content described in decisions.csv

Reply:
I can add that check before this. I kept this test to check if a single decision note can also be posted.